pkg/sos/sosServer_test.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
```

Comment:
yeah, if you can
pkg/sos/sosS(*.go) -> pkg/sos/s\1.go

Reply:
Fixed